src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -48,6 +50,24 @@ public static void UpdateBackground(this AView nativeView, IView view, Drawable?
 				nativeView.Background = paint!.ToDrawable();
 		}
 
+		public static void UpdateFlowDirection(this AView nativeView, IView view)
+		{
+			if (view.FlowDirection.IsRightToLeft())
+			{
+				nativeView.LayoutDirection = ALayoutDirection.Rtl;
+
+				if (nativeView is TextView textView)
+					textView.TextDirection = ATextDirection.Rtl;
```

Comment:
Isn't HorizontalTextAlignment different from FlowDirection? You could have a Label with RTL FlowDirection and Arabic text for example but with HorizontalTextAlignment set to "End".

Reply:
Yes, text alignment is different than flow direction. But this method doesn't affect text alignment.

Reply:
Oh, sorry, I misunderstood 😅